lib/log_file_finder.rb

```diff
@@ -0,0 +1,22 @@
+def all_main_output_logs(root_dir_name)
+  []
+end
+
+class LogFileFinder
+
+  def initialize(root_log_dir)
+    @root_log_dir = root_log_dir
+    if not Dir.exist?(root_log_dir)
```

Comment:
What do you prefer, this "if not" statement or something like `raise(...) unless Dir.exist>(root_log_dir)`